src/lib/mdns/Advertiser_ImplMinimalMdns.cpp

```diff
@@ -596,15 +637,87 @@ FullQName AdvertiserMinMdns::GetCommisioningTextEntries(const CommissionAdvertis
         sprintf(txtVidPid, "VP=%d", params.GetVendorId().Value());
     }
 
-    char txtPairingInstrHint[128];
-    if (params.GetPairingInstr().HasValue() && params.GetPairingHint().HasValue())
+    char txtDeviceType[32];
```

Comment:
We should start adding constants for these string sizes.

Reply:
Currently I'm just trying to be a good citizen and follow the conventions already used in this file. If there is a great example elsewhere, please point me to it and I will follow.